src/api.js

```diff
@@ -10,19 +10,19 @@ const searchRecipes = async (term) => {
   return response.data.data;
 };
 
-const getRecipe = async (id) => {
+const selectRecipe = async (id) => {
   const response = await axios.get(
     `https://forkify-api.herokuapp.com/api/v2/recipes/${id}`
   );
   return response.data.data.recipe;
 };
 
-const getBookmarks = async () => {
+const fetchBookmarks = async () => {
```

Comment:
[super nit] I usually name these function according to the CRUD operation names, so this can still be `getBookmarks`